ios/Helium.xcodeproj/project.pbxproj

```diff
@@ -653,6 +657,24 @@
 			shellScript = "diff \"${PODS_PODFILE_DIR_PATH}/Podfile.lock\" \"${PODS_ROOT}/Manifest.lock\" > /dev/null\nif [ $? != 0 ] ; then\n    # print error to STDERR\n    echo \"error: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.\" >&2\n    exit 1\nfi\n# This output is used by Xcode 'outputs' to avoid re-running this script phase.\necho \"SUCCESS\" > \"${SCRIPT_OUTPUT_FILE_0}\"\n";
 			showEnvVarsInLog = 0;
 		};
+		E62379C46CED8449C718F55C /* [CP] Prepare Artifacts */ = {
```

Comment:
What is this for?

Reply:
I guess maybe just from one of the updates?

Reply:
Yeah im not sure about these changes. I think it might be related to why my build is failing. I think it has to do with the flipper update

Reply:
When I removed flipper, xcode removed all of this